finish/README.txt

```diff
@@ -25,50 +23,42 @@ BankService.java file also contains code that automatically fails the first 2
 requests that are made to the checkBalance service so as to demonstrate the states 
 of a circuit breaker. 
 
-The @Fallback annotation and the fallback method are in a separate file, 
-BankServiceWithFallback.java. The fallback method is invoked whenever a request 
-to the service fails or when the circuit is open. When the fallback is running 
-the method immediately returns the following message: 
-	"The last known balance of your account is $10,293". 
+The @Fallback annotation and the fallback method are demonstrated in the same source 
+file. The fallback method is invoked whenever a request to the service fails or when 
+the circuit is open. When the circuit is opened, the request is not allowed to go 
```

Comment:
When the circuit is **open** (remove the 'd' and the comma) the request is not allowed to go through **to the service** and the fallback is **immediately called** (change order of last 2 words).

Reply:
Thank you. fixed.